src/canmatrix/formats/xls.py

```diff
@@ -292,7 +294,7 @@ def dump(db, file, **options):
                 sigstyle = sty_white
                 framestyle = sty_white
         # reset signal-Array
-        signals = []
+        signals = []  # todo unused
```

Comment:
delete unused variables causing mypy failures?

Reply:
yes, delete it